### ISSUE TITLE
Problems with running invalid program path

### ISSUE THREAD

**Issue by UnmappedStack** [Dcraftbg/MinOS#9]
When messing with the MinOS shell a bit, it has mostly worked just fine, but I have noticed that when I type the path of an invalid program, while it _does_ correctly print the "Could not find command" text then it _does_ still give the next input prompt, after I type the _next_ command, even if it's a valid path, then the system just freezes. I assume that this is probably a deadlock somewhere. I have attached a screenshot.

![image](https://github.com/user-attachments/assets/570edd47-f3de-449a-9b84-df607884280a)

**Comment by Dcraftbg** [comment]
Tysm for the issue. If you want to I'll assign you the issue

**Comment by UnmappedStack** [comment]
Yeah sure I can try work on it. I have found the general area of the issue.

**Comment by Dcraftbg** [comment]
Fixed in 131f301
